src/styles/mixins.scss

```diff
@@ -0,0 +1,31 @@
+@mixin animation-minute-hand() {
+  @keyframes rotates {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
+  animation-name: rotates;
+  transform-origin: 1.5vmin 20vmin;
+  animation-timing-function: steps(60);
+  animation-iteration-count: infinite;
+}
+
+@mixin animation-second-hand() {
+  @keyframes rotates {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
Same as above, you have defined the `@keyframes` animation rule again here. Try to avoid duplicating code whenever possible. It's better to define this animation once and reuse it in both mixins.